docs/docsite/rst/dev_guide/style_guide/index.rst

```diff
@@ -252,6 +260,59 @@ The syntax is:
    .. contents::
       :local:
 
+Accessibility guidelines
+=========================
+
+Ansible documentation has a goal to be more accessible. Use the following guidelines to help us reach this goal.
+
+Images and alternative text
+  Ensure all icons, images, diagrams, and non text elements have a meaningful alternative-text description. Do not include screen captures of CLI output. Use ``code-block`` instead.
+
+  .. code-block:: text
+
+    .. image:: path/networkdiag.png
+       :width: 400
+       :alt: SpiffyCorp network diagram
+
+
+Links and hypertext
+  URLs and cross-reference links have descriptive text that conveys information about the content of the linked target. See :ref:`style_links` for how to format links.
+
+Tables
+  Tables have a simple, logical reading order from left to right, and top to bottom.
+  Tables include a header row and avoid empty or blank table cells.
+  Label tables with a descriptive title.
+
+  .. code-block:: reStructuredText
+
+    .. table:: File descriptions
+
+      +----------+----------------------------+
+      |File      |Purpose                     |
+      +==========+============================+
+      |foo.txt   |foo configuration settings  |
+      +----------+----------------------------+
+      |bar.txt   |bar configuration settings  |
+      +----------+----------------------------+
+
+
+Colors and other visual information
+  * Avoid instructions that rely solely on sensory characteristics. For example, do not use ``Click the square, blue button to continue.``
+  * Convey information by methods and not by color alone.
+  * Ensure there is sufficient contrast between foreground and background text or graphical elements in images and diagrams.
+  * Instructions for navigating through an interface make sense without directional indicators such as left, right, above, and below.
+  * Nest headers in order from H1 to H2, H3, and so forth, without skipping heading levels. See :ref:`headers_style` for the correct order.
```

Comment:
Isn't this done automatically by RST/Spinx?

Reply:
just tested and you are correct. It spits out a CRITICAL warning if the headers are out of order so to speak.